Woodall_Robert/Assignments/angular/controllers_directives/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="app">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>First Angular Page</title>
+	
+	<!-- jquery -->
+	<script src="https://code.jquery.com/jquery-2.2.4.min.js" integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
+	
+	<!-- Bootstrap -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+	
+	<script src="bower_components/angular/angular.min.js"></script>
+	<script>
+		var app = angular.module('app', []);
+		app.controller('myController', function($scope) {
+			$scope.food = [];
+			
+			$scope.addFood = function(foodInput) {
+				if (foodInput && foodInput.length >= 1) {				
+					$scope.food.push(foodInput);
+					$scope.foodInput = '';
+				}
+			}
+			
+			$scope.printToConsole = function() {
+				console.log('ng-submit invoked...');
+			}
+		});
+	</script>
+</head>
+<body>
+	<div class="container" ng-controller="myController">
+		<h3>Getting to know controllers and directives!</h3>
+		<hr>
+		<div class="row">
+			<div class="col-md-6">
+				<form ng-submit="printToConsole()">
+					<label>Favorite Food: </label>
+					<input type="text" ng-model="foodInput">
+					<input type="submit" value="Add Food" ng-click="addFood(foodInput)">
```

Comment:
I'm guessing you know this now, but no need to pass in foodInput, since it's now "scoped" when you ng-model'd it.